project/app/api/database.py

```diff
@@ -58,135 +92,158 @@ def close(self):
         self.connection.close()
 
```

Comment:
Where you planning on continuing this function? Or are you expecting the input to be already set up in SQL code? as of right now its taking the input directly from the function input and placing that in the cursor.execute(). 

A little more notation is needed to understand what youre doing here.